src/modules/UI/scenes/TermsOfService/TermsOfService.ui.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import React, { Component } from 'react'
+import { View, WebView } from 'react-native'
+
+import Gradient from '../../components/Gradient/Gradient.ui.js'
+import SafeAreaView from '../../components/SafeAreaView'
+import { styles } from './TermsOfServiceStyle.js'
+
+const WEB_URI = 'https://edge.app/terms-of-service/'
```

Comment:
This is the wrong URI. Please get the URI from Mandie which does not have a header or footer.